app/containers/HomeContainer/index.js

```diff
@@ -4,6 +4,7 @@ import { connect } from 'react-redux';
 import { createStructuredSelector } from 'reselect';
 import { compose } from 'redux';
 import _ from 'lodash';
+import get from 'lodash/get';
```

Comment:
this is a double import. 

so basically if shouldn't be importing lodash twice. 


so let's do this. 

check the usages of `_` to find out which are the different methods we are using from lodash
then remove 
```
import _ from 'lodash'
```

and replace with something like this 
```
import get from 'lodash/get'
import debounce from 'lodash/debounce'
```

Reply:
Now i will used single in place of multiple lodash.